tiatoolbox/tools/patchextraction.py

```diff
@@ -107,18 +164,23 @@ def __init__(
         patch_size,
         resolution=0,
         units="level",
-        stride=1,
+        stride=None,
     ):
         super().__init__(
             input_img=input_img,
             patch_size=patch_size,
             resolution=resolution,
             units=units,
         )
-        self.stride = stride
+        if stride is None:
+            self.stride = self.patch_size
+        else:
+            if isinstance(stride, (tuple, list)):
+                self.stride = (int(stride[0]), int(stride[1]))
+            else:
+                self.stride = (int(stride), int(stride))
```

Comment:
Similar to above

Reply:
Refer to the comment in test section, if we dont intend to support for not-uniform stride, then better to force input to be just int.

Reply:
Fixed in these commits https://github.com/TIA-Lab/tiatoolbox/pull/97/commits/f85a61e9650b7dad8ffa73f6eafd37c5d18e5807 and https://github.com/TIA-Lab/tiatoolbox/pull/97/commits/d52f278d91a027a668335ab74b1a443bd76185ec